be/CMakeLists.txt

```diff
@@ -186,12 +185,18 @@ if (WITH_HDFS)
     set_target_properties(hdfs PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/libhdfs.a)
 endif()
 
-add_library(libevent STATIC IMPORTED)
-set_target_properties(libevent PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/libevent.a)
-
 add_library(crypto STATIC IMPORTED)
 set_target_properties(crypto PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/libcrypto.a)
 
+if (WITH_AWS)
```

Comment:
How about `WITH_AWS_S3`

Reply:
`WITH_AWS` is better, because it means to use `aws-sdk-cpp` library, which has s3, ec2 service libraries etc.